tests/JobDefinitionTest.php

```diff
@@ -33,16 +33,20 @@ public function testGettersAndSetters()
 		$jd->setClassName($expected_string);
 		$this->assertEquals($expected_string, $jd->getClassName());
 
-		$jd->setEnabled(false);
+		$jd2 = $jd->setEnabled(false);
 		$this->assertEquals(false, $jd->getEnabled());
+		$this->assertInstanceOf(JobDefinition::class, $jd2);
 
-		$jd->setMaxRunTime($expected_int);
+		$jd2 = $jd->setMaxRunTime($expected_int);
 		$this->assertEquals($expected_int, $jd->getMaxRunTime());
+		$this->assertInstanceOf(JobDefinition::class, $jd2);
 
-		$jd->setRunTime($expected_string);
+		$jd2 = $jd->setRunTime($expected_string);
 		$this->assertEquals($expected_string, $jd->getRunTime());
+		$this->assertInstanceOf(JobDefinition::class, $jd2);
 
-		$jd->setInterval($expected_int);
+		$jd2 = $jd->setInterval($expected_int);
 		$this->assertEquals($expected_int, $jd->getInterval());
+		$this->assertInstanceOf(JobDefinition::class, $jd2);
```

Comment:
It might be safer to use [`assertSame()`](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertSame)